core/soc/timer.go

```diff
@@ -50,9 +50,8 @@ func (lp *Timer) DemandActive() bool {
 		return false
 	}
 
-	se := lp.SocEstimator()
-	if se == nil {
-		lp.log.WARN.Printf("target charging: not possible")
+	if lp.Time.IsZero() {
+		lp.active = false
```

Comment:
warum muss die Hilfsvariable hier gesetzt werden? Das riecht nach einem schwer zu durchschauenden Fehler in der Logik.

Reply:
ist hier vermutlich überflüssig, da hast du recht. Teste ich.